server/cloudformation/ecs.go

```diff
@@ -95,6 +95,9 @@ func newECSServiceProvisioner(resource *ECSServiceResource) *provisioner {
 }
 
 func (p *ECSServiceResource) Create(ctx context.Context, req customresources.Request) (string, interface{}, error) {
+	// TODO: how to make this a debug log level?
+	fmt.Printf("%+v\n", req)
```

Comment:
```suggestion
	fmt.Printf("%+v\n", *req.PropagateTags)
```

I think you need to do that. That'll print the actual string referenced by req.PropagateTags.